### PR TITLE
Fix https://github.com/riot/riot/issues/2931

### PR DESCRIPTION
**Pull request by kachurun**
Fix for https://github.com/riot/riot/issues/2931

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -137,16 +138,12 @@ export function compile(source, opts = {}) {
...
-  if (
-    // filter missing nodes
-    !sourceNode ||
-    // filter nodes without children
-    (sourceNode.nodes && !sourceNode.nodes.length) ||
-    // filter empty javascript and css nodes
-    (!sourceNode.nodes && !sourceNode.text)) {
+  const hasContent = sourceNode && (sourceNode.text || !isEmptyArray(sourceNode.nodes) || !isEmptyArray(sourceNode.attributes))
+
+  if (!hasContent) {
```
Comment (GianlucaGuarini) [inline src/index.js:146]: Could you please change this if to?

```hasContent ? curry(transformer)(sourceNode, source, meta) : result => result```

Comment (GianlucaGuarini) [reply src/index.js:146]: I prefer positive conditions to negative lookups

**Review by GianlucaGuarini — CHANGES_REQUESTED** [review]
Thank you, there is only something small to update and we can merge it

File: src/index.js (hunk 1 above)
Comment (kachurun) [reply src/index.js]: Done

**Pull request merged**